indico/modules/events/registration/client/js/form_submission/RegistrationFormSubmission.jsx

```diff
@@ -170,7 +172,7 @@ export default function RegistrationFormSubmission() {
       onSubmit={onSubmit}
       initialValues={isUpdateMode ? registrationData : initialValues}
       initialValuesEqual={_.isEqual}
-      subscription={{}}
+      subscription={{errors: true}}
```

Comment:
Are you sure that's needed? I think if you use `useFormState` in your component you can specify the subscription in there in order to just re-render your component instead of the whole form here.

Reply:
I changed it to `FormSpy` which subscribes to errors.

Reply:
You can revert the change to this line then, since the component where you added the `useFormState` will automatically re-render and it should no longer be necessary to re-render here.

Reply:
Hm, I thought I have. Guess not. 😂